other/upgrade_2-1_mysql.sql

```diff
@@ -1060,6 +1061,67 @@ ALTER TABLE {$db_prefix}members
 	DROP notify_announcements;
 ---#
 
+---# Upgrading auto notify setting
+---{
+$_GET['a'] = isset($_GET['a']) ? (int) $_GET['a'] : 0;
+$step_progress['name'] = 'Upgrading auto notify setting';
+$step_progress['current'] = $_GET['a'];
+
+$limit = 100000;
+$is_done = false;
+
+$request = $smcFunc['db_query']('', 'SELECT COUNT(*) FROM {db_prefix}themes WHERE variable = \'auto_notify\'');
+list($maxMembers) = $smcFunc['db_fetch_row']($request);
+$smcFunc['db_free_result']($request);
+
+while (!$is_done)
+{
+	nextSubStep($substep);
+	$inserts = array();
+
+	// This setting is stored over in the themes table in 2.0...
+	$request = $smcFunc['db_query']('', '
+		SELECT id_member, value
+		FROM {db_prefix}themes
+		WHERE variable = \'auto_notify\'
```

Comment:
Missing the use of string replacement {string:auto_notify} .. array(auto_notify => 'auto_notify')

Reply:
Done